Source/CarthageKit/RemoteVersion.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import ReactiveSwift
+import ReactiveTask
+import Result
+import Tentacle
+
+/// Synchronously returns the newest Carthage release version,
```

Comment:
Is the doc mentioning "Carthage" still relevant?

Reply:
Yeah, very good point. I updated the docs.